scripts/generator.py

```diff
@@ -3,6 +3,7 @@
 import os
 import schema_reader
 import yaml
+import json
```

Comment:
Nit: Please remove this new import, as it's no longer needed :-)

Reply:
ah good point